### PR TITLE
[blog] add 5th birthday blog post

### PR DESCRIPTION
**Pull request by flynnduism**
Signed-off-by: flynnduism <dev@ronan.design>

### CODE REVIEW

File: content/en/blog/2018-10-19-helm-turns-five.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+title: " Helm Turns 5, and GitHub Gives the Gift of Charts"
+slug: "helm-turns-five"
+authorname: "Matt Butcher & Matt Farina"
+author: "@technosophos & @mattfarina"
+authorlink: "https://helm.sh"
+date: "2020-10-16"
```
Comment (flynnduism) [inline content/en/blog/2018-10-19-helm-turns-five.md:7]: This is set to today for preview purposes. 
Will adjust to the 19th once lgtm ✅

**Comment by bridgetkromhout** [comment]
Hmm, the formatting for the initial commit does not look right in the preview: https://deploy-preview-901--helm-merge.netlify.app/blog/ - maybe just set the preview to end right after "born"?

File: content/en/blog/2018-10-19-helm-turns-five.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    initial add
+```
+
+[This commit](https://github.com/helm/helm-classic/commit/ecad6e2ef9523a0218864ec552bbfc724f0b9d3d) can be found on the helm-classic Git repository where the codebase for Helm v1 is located. This is the original Helm, before it merged with Deployment Manager and was folded into Kubernetes. This is where it all began.
+
+Since day one, the Helm project has relied upon GitHub for source control, pull request management, and issue tracking. As a graduated CNCF project, the Helm org now manages dozens of GitHub repositories.
+
+But when it came to hosting the packaged charts, we stored them in an object storage bucket hosted on Google Cloud. This historical decision reflects that at that time Google was one of the principal contributors to Helm.
+
+Recently, Google’s time of supporting the official Helm chart repository has come to a close. We are greateful for Google’s hosting the Helm chart repository these last few years. But this event has given us an opportunity to further integrate our chart development pipeline with GitHub.
```
Comment (bridgetkromhout) [inline content/en/blog/2018-10-19-helm-turns-five.md:28]: Typo crept in; `greateful` should be `grateful`.

**Comment by mattfarina** [comment]
@flynnduism can you update the date on this?

**Review by bridgetkromhout — APPROVED** [review]

**Pull request merged**